src/spark/primitives.h

```diff
@@ -49,6 +49,7 @@ struct CSparkMintMeta
         READWRITE(memo);
         READWRITE(serial_context);
         READWRITE(type);
+        coin.setParams(spark::Params::get_default());
```

Comment:
I think this should be done only when __reading__ coin from the stream, writing shouldn't modify anything

Reply:
okay, agree, will change